docs/it-manual/sre-playbook/monitoring.md

```diff
@@ -57,3 +60,68 @@ To import this pre-built dashboard:
 5. Paste [the JSON here](https://gist.githubusercontent.com/dkatzz/a7048a6db9a2cbb14ef96f744f5fd964/raw/f65295421e47782cc55eecdbf70653d676121070/grafana_json) into the "Import via panel JSON" 
 7. Click "Load"
 8. Fill in the details for the imported dashboard, selecting your CiviForm prometheus instance for the data source
+
+## Additional AWS monitoring
+
+In addition to the server metrics provided by Prometheus, there are some additional places within AWS you can go to see metrics.
+
+### CloudWatch
+
+#### Dashboards
+
+CloudWatch has some default dashboards that allow you to see graphs with metrics on different parts of the deployment. Not all of these are relevant, but these can be helpful in seeing CPU utilization in RDS (the database) and Elastic Container Service, as well as requests to the Application ELB (load balancer), metrics about S3, etc.
+![](<../../.gitbook/assets/aws-cloudwatch-dashboards.png>)
+
+#### Alarms
+
+Some alarms are configured by default through terraform, which includes the following: 
+- ECS:
+  - High CPU alarm
+    - Related variables: `ECS_MAX_CPU_THRESHOLD`, `ECS_MAX_CPU_EVALUATION_PERIOD`, `ECS_MAX_CPU_PERIOD`, `ECS_SCALE_TARGET_MAX_CAPACITY`
+  - Low CPU alarm
+    - Related variables: `ECS_MIN_CPU_THRESHOLD`, `ECS_MIN_CPU_EVALUATION_PERIOD`, `ECS_MIN_CPU_PERIOD`, `ECS_SCALE_TARGET_MIN_CAPACITY`
+- RDS:
+  - High CPU alarm
+    - Related variables: `RDS_CREATE_HIGH_CPU_ALARM`, `RDS_MAX_CPU_UTILIZATION_THRESHOLD`
+  - High disk queue depth alarm
+    - Related variables: `RDS_CREATE_HIGH_QUEUE_DEPTH_ALARM`, `RDS_DISK_QUEUE_DEPTH_HIGH_THRESHOLD`
+  - Low disk space alarm
+    - Related variables: `RDS_CREATE_LOW_DISK_SPACE_ALARM`, `RDS_DISK_FREE_STORAGE_LOW_THRESHOLD`
+  - Low freeable memory alarm
+    - Related variables: `RDS_CREATE_LOW_MEMORY_SPACE_ALARM`, `RDS_LOW_MEMORY_THRESHOLD`
+  - Maximum used transaction IDs alarm
```

Comment:
**suggestion (non-blocking):** add a parenthetical explaining what this is

Reply:
This got removed when I removed the engine variable to disable the alarm, but I will add it back later and setup a boolean for this to be enabled / disabled as well.